pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.ecocode</groupId>
     <artifactId>ecocode-java-plugin</artifactId>
-    <version>1.5.1-SNAPSHOT</version>
+    <version>1.5.2-SNAPSHOT</version>
```

Comment:
same remark like docker-compose modification

Reply:
I will do the same revert

Reply:
ok